EventFilter/CTPPSRawToDigi/src/RawDataUnpacker.cc

```diff
@@ -425,3 +433,57 @@ int RawDataUnpacker::ProcessVFATDataParallel(const uint16_t *buf, unsigned int m
 
   return wordsProcessed;
 }
+
+//----------------------------------------------------------------------------------------------------
+
+int RawDataUnpacker::ProcessOptoRxFrameSampic(const word *buf, unsigned int frameSize, TotemFEDInfo &fedInfo, SimpleVFATFrameCollection *fc) const
+{
+  
+  
+  unsigned int OptoRxId = (buf[0] >> 8) & 0xFFF;
+//   std::cout<< "Processing sampic frame: OptoRx " << OptoRxId << "  GOH " << gohIdx << "  Idx " << fiberIdx << "   framesize: " << frameSize << std::endl;
```

Comment:
Please remove commented out lines (here and below), or replace them with some LogSomething(...) if you want to keep them